lib/matchers/Concat.ts

```diff
@@ -56,6 +56,13 @@ export type MatchStep = Matcher | MatchVeto | ContextComputation;
 
 const methodsOnEveryMatchingLogic = ["$id", "matchPrefix", "canStartWith", "requiredPrefix"];
 
+export type ConcatDefinitions = any; // maybe we can tighten this. For now, giving it a name
+export interface ConcatEnough extends MatchingLogic {
```

Comment:
Typedoc on this would be good

Reply:
I'm not crazy about the name but am struggling to think of a better one, given the concrete class uses the logical name. While it's a little odd to have a class with a shorter name than the interface, I think I'd use `Concatenation` for the new interface, as it is the technically correct term. Also this interface isn't often referenced in user code.

Reply:
OK that makes sense! I knew you'd find a better name.